docker/devnet/booster-http/entrypoint.sh

```diff
@@ -10,4 +10,4 @@ echo $MINER_API_INFO
 echo $BOOST_API_INFO
 
 echo Starting booster-http...
-exec booster-http run --serve-files=true --api-boost=$BOOST_API_INFO --api-fullnode=$FULLNODE_API_INFO --api-storage=$MINER_API_INFO --tracing
+exec booster-http run --serve-files=true --api-boost=$BOOST_API_INFO --api-fullnode=$FULLNODE_API_INFO --api-storage=$MINER_API_INFO --tracing --nitro-enabled --nitro-endpoint=host.docker.internal:4007/api/v1
```

Comment:
Longer term we should probably consider overriding these with env vars like lotus does, instead of hardcoding them in the entrypoint.